.gitignore

```diff
@@ -56,6 +56,15 @@ junit.xml
 !/.eslintrc.json
 .jsii
 tsconfig.json
+test/cdk-lib/bedrock/integ-tests/.tmp
+test/cdk-lib/bedrock/integ-tests/prompts.integ.snapshot/asset.*
```

Comment:
I don't see a convention adopted for the outputs during integration testing at https://docs.aws.amazon.com/cdk/api/v2/docs/integ-tests-alpha-readme.html, but which of these may allow a convention to adopt if these are added to other namespaces? I'm also curious where the `manifest.json` and `tree.json` come from?

```
test/cdk-lib/**/.tmp
test/cdk-lib/**/asset.*
test/cdk-lib/**/cdk.out
```

Reply:
Most of these modifications are done automatically by projen. An `AwsCdkConstructLibrary` has by default `integrationTestAutoDiscover: true` which means that as soon as an `integ.ts` file is detected, appropriate commands for deploying/destroyin/asserting/... are created under package.json as well as modifications to files such as `.gitignore` and some other files . You can try to add a file named `my-test.integ.ts`, then run build and see how the related files change.